src/popup/components/LatestTransactionsCard.vue

```diff
@@ -113,21 +112,28 @@ export default defineComponent({
       ))
         .sort(defaultTransactionSortingCallback)
         .slice(0, DASHBOARD_TRANSACTION_LIMIT);
-    }
-
-    // This replaces it poling
-    watch(balances, updateData);
 
-    onMounted(async () => {
-      await updateData();
       isLoading.value = false;
-    });
+    }
+
+    // To avoid unnecessary data transfers instead of constant polling
+    // we are fetching the transactions only if the total balance of the accounts changes.
```

Comment:
We need also to watch changes of token balances for every account, in order to cover also fungible token transactions.

Reply:
#1928

Reply:
Your PR is fixing this: https://github.com/aeternity/superhero-wallet/pull/1928/files
We'll merge both of them in order to fix the whole problem.